modules/ai/module.json

```diff
@@ -46,6 +51,18 @@
 			"system": "dnd5e"
 		}
 	],
+	"relationships": {
+		"systems": [
+			{
+				"id": "dnd5e",
+				"type": "system",
+				"manifest": "https://github.com/foundryvtt/dnd5e/releases/download/release-2.0.3/system.json",
```

Comment:
Shouldn't the manifest for the dependency always point to the latest, otherwise foundry might install 2.0.3 version even though it's outdated?